projects/vertical-components/src/lib/tabs/tab/tab.component.scss

```diff
@@ -0,0 +1,46 @@
+:host {
+  .mat-ripple {
+    padding: 0.53125rem 0.625rem;
+  }
+}
+
+@mixin tab($primary) {
+  .vertical-tab {
+    align-items: center;
+    box-shadow: none;
+    box-sizing: border-box;
+    color: mat-contrast($primary, 50); // TODO: Is this the right color? In Design: #424242
+    cursor: pointer;
+    display: inline-flex;
+    font-size: 0.875rem;
+    font-weight: 300;
+    height: 2.0625rem;
+    justify-content: center;
+    letter-spacing: 0.3px;
+    line-height: 1rem;
+    min-width: 3.125rem;
+    user-select: none;
+  }
+
+  .vertical-tab .mat-ripple-element {
+    background-color: rgba(mat-color($primary, 900), 0.1);
+  }
+
+  .vertical-tab:hover {
+    box-shadow: inset 0 -1px 0 rgba(mat-contrast($primary, 50), 0.12);
+    background-color: rgba(mat-contrast($primary, 50), 0.04);
+  }
+
+  .vertical-tab-active {
+    /* Using box-shadow instead of border-bottom to inset the indicator */
+    box-shadow: inset 0 -2px 0 mat-color($primary, 900);
+    color: mat-color($primary, 900);
+    font-weight: 500;
+  }
+
+  .vertical-tab-active:hover {
+    background-color: rgba(mat-color($primary, 700), 0.04);
+    // TODO: DRY, find a way to fix it
+    box-shadow: inset 0 -2px 0 mat-color($primary, 900);
```

Comment:
Do you really need this line?

Reply:
Yes, because if I don't include it, it takes the hover-class of "vertical-tab" , which I don't want on the active tab. Is there a way to implement "don't use the vertical-tab:hover-class, if the element also has the vertical-tab-active-class"? I don't want to use "!important" on the box-shadow, because it is not considered "clean code".

Reply:
Worked!